strapi/strapi
# [TypeScript] Add typescript loader in the webpack config

### What does it do?

It adds a loader to parse `ts|tsx` file so we can enable TypeScript in the codebase. It also adds `tsconfig.json` files for all the admin part of each plugin of the monorepo.


### Why is it needed?

To support TypeScript

### How to test it?

1. Run the getstarted app in dev mode `yarn develop --watch-admin` 
2. In the myplugin/admin/src/** create a component in ts
3. Check that it is loaded correctly
4. Check that the app is building correctly
5. In a plugin create a component in TS and import it
6. Check that it is still loaded correctly.
